content/neuvector-prime/azure/contents.lr

```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+The existing AKS cluster must be on a supported PAYG platform with its Neuvector configuration. To manually upgrade the recent version via the "az k8s-extension update" command.
+
+Execute the following commands in Cluster Cloud Shell:
 
 -  ---
    **NOTE** if cluster is deployed in westcentralus location, execute the following command.
            This command will also work on other regions once the feature is rolled out to other regions.
 
 -  ---
 
+To see the most recent published extension version to be upgraded.
```

Comment:
this will show the latest version available? maybe rephrase as:
"To see the most recently published extension version available as an upgrade."